test/lib/core_assertions.rb

```diff
@@ -694,8 +694,7 @@ def assert_join_threads(threads, message = nil)
         if !errs.empty?
           msg = "exceptions on #{errs.length} threads:\n" +
             errs.map {|t, err|
-            "#{t.inspect}:\n" +
-              RUBY_VERSION >= "2.5.0" ? err.full_message(highlight: false, order: :top) : err.message
+            "#{t.inspect}:\n" + err.full_message(highlight: false, order: :top)
           }.join("\n---\n")
```

Comment:
I realize that this isn't the fault of this PR, but the indentation of these lines look odd. Could you fix it?